stripe-core/src/Web/Stripe/Types.hs

```diff
@@ -535,13 +512,98 @@ data DefaultCard = DefaultCard { getDefaultCard :: CardId }
 ------------------------------------------------------------------------------
 -- | `SubscriptionId` for a `Subscription`
 newtype SubscriptionId = SubscriptionId { getSubscriptionId :: Text }
-    deriving (Read, Show, Eq, Ord, Data, Typeable)
+    deriving (Read, Show, Eq, Ord, Data, Typeable, FromJSON)
+
+
+data Session = Session {
+      sessionId                         :: SessionId
+    , sessionCancelUrl :: CancelUrl
+    , sessionSuccessUrl :: SuccessUrl
+    , sessionLivemode :: Bool
+    , sessionClientReferenceId :: Maybe ClientReferenceId
+    , sessionCustomerEmail :: Maybe CustomerEmail
+    , sessionBillingAddressCollection :: Maybe TODO
+    , sessionDisplayItems :: Maybe [TODO]
```

Comment:
@NorfairKing Do you remember where you got `sessionDisplayItems` from? I can't see anything about display items in [the API reference][0].

[0]: https://stripe.com/docs/api/checkout/sessions